.travis.yml

```diff
@@ -27,15 +27,25 @@ before_install:
   - sudo ldconfig
   - cd ..
   - sudo apt-get update
-  - sudo apt-get install --assume-yes apache2-mpm-prefork apache2-bin apache2 apache2-prefork-dev apache2-utils
+  - sudo apt-get install --assume-yes apache2-mpm-prefork apache2-bin apache2 apache2-prefork-dev apache2-utils libapache2-mod-perl2
+  - sudo apt-get install --assume-yes libtool libpcre3-dev libexpat1-dev libapache2-modsecurity
```

Comment:
Am I missing something, or are you installing ModSecurity twice? Once through `apt-get [...] libapache2-modsecurity` and once through the `git clone`? I'd guess we'd only want the former, and not the latter, because then we'd be depending on the master branch of an external repo to pass our tests.

Reply:
I removed the "standard" install of ModSecurity that was only used to ensure all depedencies were installed.
We need to compile it ourself as the Ubuntu build is a "simplified" version lacking the --enable-request-early flag.
We could link to a specific version to be safer...